.github/workflows/test-e2e.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
```

Comment:
No breaking changes called out in https://github.com/actions/setup-go/releases/tag/v5.0.0